### ISSUE TITLE
"Feature Request: Add User-Configurable 'Custom' Field for Model Version Selection in User Interface"

### ISSUE THREAD

**Issue by un33k** [ztjhz/BetterChatGPT#616]
Today we got: `On Wednesday, October 2nd, the default version of GPT-4o will be updated to the latest GPT-4o model, gpt-4o-2024-08-06.``

To enhance the feature for updating models via the user interface sooner, the addition of a user-configurable "custom" field alongside the dropdown menu for models can be useful. By inputting the desired model version directly into the custom field, users could access the update without waiting the full three weeks. This customization feature would provide more flexibility and convenience in selecting specific model versions tailored to individual needs.

**Comment by bitclick** [comment]
yes, i second that.  i got an email that i can use `o1-preview` now but cannot choose it in the dropdown in betterchatgpt..
